commands/wordpressComposerUpdate.js

```diff
@@ -10,52 +10,133 @@ import {
+      asyncExecOptions,
+    )
+    writeInfo(`running: ${requireCommand}`)
+    console.log(requiredOutput.stdout || requiredOutput.stderr)
+  }
+}
+
+/**
+ * @param {object} asyncExecOptions
+ * @param {string} asyncExecOptions.stdio
+ * @param {string} asyncExecOptions.cwd
+ * @return {Promise<void>}
+ */
+async function validateComposerFile(asyncExecOptions) {
+  const composerValidateOutput = await asyncExec(
+    "composer validate --no-check-all --strict --no-interaction --ansi",
+    asyncExecOptions,
+  )
+  console.log(composerValidateOutput.stdout)
+}
+
+/**
+ * @returns {object}
+ */
+function getComposerJson() {
+  return JSON.parse(fs.readFileSync(`${process.env.PWD}/composer.json`, "utf8"))
+}
+
+/**
+ * @returns {Promise<void>}
+ */
+export default async function wpUpdate() {
   const asyncExecOptions = {
     stdio: "inherit",
     cwd: process.env.PWD,
   }
 
   const parsedPhpVersion = semver.parse(versions.php, {})
   const phpVersion = `>=${parsedPhpVersion.major}.${parsedPhpVersion.minor}`
+  const minimumStability = "dev"
+  const preferStable = true
 
   try {
-    const composerValidateOutput = await asyncExec(
-      "composer validate --no-check-all --strict --no-interaction --ansi",
-      asyncExecOptions,
-    )
-    console.log(composerValidateOutput.stdout)
-
-    const composerJson = JSON.parse(
-      fs.readFileSync(process.env.PWD + "/composer.json", "utf8"),
-    )
-
-    for (const [requiredComposerPackage] of Object.entries(
-      composerJson.require,
-    )) {
-      let requireCommand = ""
-      if (requiredComposerPackage === "php") {
-        requireCommand = `composer require "${requiredComposerPackage}:${phpVersion}" --with-all-dependencies --no-interaction  --ansi`
-      }
-      if (requiredComposerPackage !== "php") {
-        requireCommand = `composer require ${requiredComposerPackage} --with-all-dependencies --no-interaction --no-progress --ansi`
-      }
-      let requiredOutput = await asyncExec(requireCommand, asyncExecOptions)
-      writeInfo("running: " + requireCommand)
-      console.log(requiredOutput.stdout || requiredOutput.stderr)
-    }
-
-    for (const [requiredDevComposerPackage] of Object.entries(
-      composerJson["require-dev"],
-    )) {
-      const requireCommand = `composer require ${requiredDevComposerPackage} --dev --with-all-dependencies --no-interaction --no-progress --ansi`
-      let requiredOutput = await asyncExec(
-        `composer require ${requiredDevComposerPackage} --dev --with-all-dependencies --no-interaction --no-progress --ansi`,
-        asyncExecOptions,
-      )
-      writeInfo("running: " + requireCommand)
-      console.log(requiredOutput.stdout || requiredOutput.stderr)
-    }
+    await validateComposerFile(asyncExecOptions)
+    const composerJson = getComposerJson()
+    setMinimumStability(composerJson, minimumStability)
+    setPreferredStability(composerJson, preferStable)
```

Comment:
Nu slog mig en tanke, kommer `minimumstability` bli överskrivet här? Eftersom att du läser in filen på rad 135